static-fire/trivyignore

```diff
@@ -0,0 +1,3 @@
+# Upstream registry service include several CVEs.
+# We should try to remove these each time we update.
+CVE-2022-32149
```

Comment:
Does this apply to this static-fire? This looks like it was copied from registry but is not needed here. If it's needed we should improve this comment text and make it more accurate for the static-fire project.

Reply:
Yeah, I'll remove it